src/components/FlowRunCreateForm.vue

```diff
@@ -116,14 +117,22 @@
     start: fieldRules('Start date', isRequiredIf(() => when.value === 'later')),
   }
 
+  const parameters = computed(() => {
+    if (props.flowRun) {
+      return { ...props.deployment.parameters, ...props.flowRun.parameters }
+    }
+
+    return props.deployment.parameters
+  })
```

Comment:
You can spread `undefined` into an object so you can simplify this
```suggestion
  const parameters = computed(() => {
    return { ...props.deployment.parameters, ...props.flowRun.parameters }
  })
```